crates/viewer/re_context_menu/src/actions/screenshot_action.rs

```diff
@@ -0,0 +1,78 @@
+use re_viewer_context::{
+    Item, PublishedSpaceViewInfo, ScreenshotTarget, SpaceViewId, SpaceViewRectPublisher,
+};
+
+use crate::{ContextMenuAction, ContextMenuContext};
+
+/// Space view screenshot action.
```

Comment:
```suggestion
/// View screenshot action.
```

...one of these days I'll kill all the `space`. Any moment now :smile:

Reply:
Is there an issue/motivation for this? And why not just do it? Seems like an easy search-and-replace?

Reply:
No, issue for this, no.
Many many months ago this is something that we concluded together with @nikolausWest . I think @abey79 was involved as well. The rationale is that "space view" is unnecessarily confusing over just "view" and sometimes leads to fairly lengthy names.
Ever since we slowly moved towards removing "space" from types and comments (well.. at least Antoine and me did). We never got around to doing the whole thing because it touches an enormous amount of types & code, a few of which are part of the API.